File: java-jpa/src/main/java/edu/pkch/jpa/Member.java
package edu.pkch.jpa;

import javax.persistence.*;

@Entity
@Table(name = "member")
public class Member {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column
    private Long memberNumber;

    @Column(length = 100)
    private String nickname;

    public Member() {}

    public Member(Long memberNumber, String nickname) {
        this.memberNumber = memberNumber;
        this.nickname = nickname;
    }
}

File: kotlin-jpa/src/main/kotlin/edu/pkch/jpa/Member.kt
package edu.pkch.jpa

import javax.persistence.*

@Entity
@Table(name = "member")
class Member(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long,

    @Column
    var memberNumber: Long,

    @Column(length = 100)
    var nickname: String
)